ui/src/components/SampleControls/SnapshotControl.jsx

```diff
@@ -25,19 +25,30 @@ function SnapshotControl(props) {
     return sourceScale * imageRatio;
   });
 
-  const takeSnapshot = useCallback(() => {
+  const takeSnapshot = useCallback(async () => {
     const img = document.querySelector('#sample-img');
     const fimg = new fabric.Image(img);
     fimg.scale(imageRatio);
 
     canvas.setBackgroundImage(fimg);
     canvas.renderAll();
```

Comment:
I'd be curious to know if we still need this preparation code (and corresponding cleanup code).

Reply:
They did help create a cleaner overlay between the camera image and the canvas in the background